dashboard/src/components/AppView/AppDetails.test.tsx

```diff
@@ -29,7 +29,7 @@ it("renders a deployment details", () => {
 it("renders a services details", () => {
   const services = new Map<string, IResource>();
   const svc = "foo";
-  services[svc] = {
+  services.set(svc, {
```

Comment:
What has changed here that forced you to do this?

Reply:
Hmm, I torn on this, the Map type is not a real JavaScript type (atleast in ES5) and I prefer accessing/setting as if it's a normal JavaScript object over using set/get, but that's just my preference. I don't have a strong opinion though.